index.js

```diff
@@ -4,7 +4,8 @@ const mysql = require("./dbcon.js");
 const hbs = require("handlebars");
 const exphbs = require("express-handlebars");
 const path = require("path");
-var bodyParser = require("body-parser");
+const bodyParser = require("body-parser");
+const cookieSession = require("cookie-session");
```

Comment:
I think another reason to run npm-install is for cookie-session, which we weren't using before.

Reply:
👍